libs/langchain/langchain/prompts/chat.py

```diff
@@ -261,63 +261,31 @@ def format_messages(self, **kwargs: Any) -> List[BaseMessage]:
 
 
 class ChatPromptTemplate(BaseChatPromptTemplate, ABC):
-    """Use to create flexible templated prompts for chat models.
+    """A prompt template for a chat models.
 
-    Examples:
+    Use to create flexible templated prompts for chat models.
 
-        Instantiation from role strings:
+    Examples:
 
         .. code-block:: python
 
             from langchain.prompts import ChatPromptTemplate
 
-            prompt_template = ChatPromptTemplate.from_role_strings(
-                [
-                    ('system', "You are a helpful bot. Your name is {bot_name}."),
-                    ('human', "{user_input}")
-                ]
-            )
-
-            prompt_template.format_messages(
-                bot_name="bobby",
-                user_input="Hello! What is your name?"
-            )
-
-        Instantiation from messages:
-
-        This is useful if it's important to distinguish between messages that
-        are templates and messages that are already formatted.
-
-        .. code-block:: python
-
-            from langchain.prompts import (
-                ChatPromptTemplate,
-                HumanMessagePromptTemplate,
-                SystemMessagePromptTemplate,
-            )
-
-            from langchain.schema import AIMessage
-
-            prompt_template = ChatPromptTemplate.from_messages(
-                [
-                    SystemMessagePromptTemplate.from_template(
-                        "You are a helpful bot. Your name is {bot_name}."
-                    ),
-                    AIMessage(content="Hello!"), # Already formatted message
-                    HumanMessagePromptTemplate.from_template(
-                        "{user_input}"
-                    ),
-                ]
-            )
+            template = ChatPromptTemplate.from_messages([
+                ("system", "You are a helpful AI bot. Your name is {name}."),
+                ("human", "Hello, how are you doing?"),
+                ("assistant", "I'm doing well, thanks!"),
```

Comment:
this should be "ai" now?

Reply:
yes! fixing in a follow up pr: https://github.com/langchain-ai/langchain/pull/8424/files